lib/cisco_node_utils/cmd_ref/bridge_domain.yaml

```diff
@@ -0,0 +1,44 @@
+# bridge_domain
+# bridge_domain feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k]
+
+all_bds:
+  multiple: true
+  get_command: "show bridge-domain"
+  get_value: '/^Bridge-domain (\d+) /'
+
+create:
+  set_value: "system bridge-domain add %s ; bridge-domain %s"
```

Comment:
As mentioned before, please use the key-val technique for all yaml entries. Also, don't use the semi-colon, use: `['system bridge-domain add %s', 'bridge-domain %s']

This raises another question: You're doing an add operation here instead of a replace. Is that going to cause idempotency issues (if the current state doesn't match the manifest state)?